tailwind.config.js

```diff
@@ -0,0 +1,27 @@
+/** @type {import('tailwindcss').Config} */
+module.exports = {
+  content: ['./src/**/*.{js,jsx,ts,tsx}'],
+  theme: {
+    extend: {
```

Comment:
I don't want you to extend tailwind default color palette, but override it with only a few - we want to leave as little space for mistakes as possible

If you'll need to have a different color in your app later on, you'll add this in here and then use it.

Reply:
done